src/test/java/org/broadinstitute/hellbender/engine/GATKToolUnitTest.java

```diff
@@ -33,11 +33,8 @@
 import org.testng.annotations.Test;
 
 import java.io.File;
-import java.util.Arrays;
-import java.util.Iterator;
+import java.util.*;
 import java.io.IOException;
-import java.util.List;
-import java.util.Set;
 
 public final class GATKToolUnitTest extends GATKBaseTest {
```

Comment:
These tests for file expansion suppression should be in `CommandLineProgramUnitTest` instead of here, since `GATKToolUnitTest` is for testing `GATKTool`-specific features.

Reply:
done.